dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -46,18 +48,19 @@
      * @param chromosomeStart Start coordinate in chromosome, null if the contig is not fully mapped to a chromosome
      * @param chromosomeEnd End coordinate in chromosome, null if the contig is not fully mapped to a chromosome
      */
-    public SubSnpCoreFields(long ssId, Long rsId, int snpOrientation, String contig, int contigStart, int contigEnd,
-                            int contigOrientation, String chromosome, Integer chromosomeStart, Integer chromosomeEnd) {
+    public SubSnpCoreFields(long ssId, Integer rsId, int snpOrientation, String contig, int contigStart, int contigEnd,
+                            int contigOrientation, String chromosome, Long chromosomeStart, BigDecimal chromosomeEnd) {
```

Comment:
You were right @jmmut , using 4 different data types to represent the coordinate parameters looks quite messy... Maybe we could store them all as `long` to make them more consistent while not losing the highest values?

Reply:
agreed offline to use integers, because that's the actual type in the database, and the query operations won't make big changes